devito/arch/compiler.py

```diff
@@ -737,6 +737,54 @@ def __init__(self, *args, **kwargs):
             warning("Running on Intel KNL without OpenMP is highly discouraged")
 
 
+class OneapiCompiler(IntelCompiler):
+
+    def __init__(self, *args, **kwargs):
+        super(OneapiCompiler, self).__init__(*args, **kwargs)
+
+        language = kwargs.pop('language', configuration['language'])
+        platform = kwargs.pop('platform', configuration['platform'])
+
+        self.cflags.append("-qopt-zmm-usage=high")
```

Comment:
That's oddly specific, this works on any platform?

Reply:
no, indeed. 

Can you please `git cherry-pick` (and adjust?) this commit of mine from #2114 ? 
https://github.com/devitocodes/devito/pull/2114/commits/4afe4f849027d4720af19444a115a6e96fb84ae5

Reply:
I do not think it is needed anymore